docs/docs/cmd/spp/model/model-list.mdx

```diff
@@ -0,0 +1,278 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# m365 spp model list
+
+Retrieve the list of SharePoint Premium unstructured document processing models on the content center site
```

Comment:
Let's use the shorter description variant.

Reply:
Done